app/main.js

```diff
@@ -260,7 +271,7 @@ function checkForWin() {
 function checkForLoss() {
     if (lives == 0) {
         resetGame();
-        drawMessage(youWin);
+        drawMessage(gameOver);
```

Comment:
I was confused by the lives == 0, youWin message before. Why was it that way before and now it is gameOver?

Reply:
It was just a mistake. The `gameOver` variable should be in there like it is now.